.gitignore

```diff
@@ -59,3 +59,6 @@ docker-compose.override.yml
 
 # Module upload folder
 /web/modules/local
+
+# Danish translations folder
+/config/sync/language/da
```

Comment:
Consider deleting the contents of this folder if we do not care about the contents anymore because [translations now live in translations on GitHub Pages](https://github.com/danskernesdigitalebibliotek/dpl-cms/blob/gh-pages/translations/da.config.po#L15-L17).